modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -3,208 +3,125 @@ ifndef::env-site,env-github[]
+. Select *Resource Allocation*.
 +
-If you don’t see the *Add Keystore* button, ensure that you are viewing your secret group in edit mode. Return to *Secret Groups* and then select the *Edit* next to your group.
-
-.. Enter a name for your keystore.
+The number of inbound load balancer replicas is determined by the number of controller nodes configured in the RTF. A minimum of 2 replicas is required for production environments, and these replicas only run on the controller VMs. Refer to xref:manage-index.adoc[Manage Runtime Fabric] for detailed information.
 +
-This name identifies the keystore when you add it to your TLS context below.
+*Shared Mode* is automatically selected for the ingress configuration if no dedicated mode is available.
 
-.. Under *Certificate File*, select *Choose File* and then select your `.pem` certificate` file.
-.. Under *Key File*, select *Choose File* and then select your `.pem` key file.
-.. Enter your key passphrase.
-.. Optionally, upload a separate certificate file containing a CA path.
-.. Select *Save*.
+.. Specify the minimum number of cores for each replica of the internal load balancer. Note that Runtime Fabric can use up to the maximum number shown if they are available. Transport Layer Security (TLS) termination is computationally expensive, so make sure to allocate enough CPU to increase throughput and decrease latency.
+.. Specify the minimum memory for each replica of the internal load balancer. Note that Runtime Fabric can use up to the maximum amount shown if it is available.
+. Configure a TLS context.
 +
-[NOTE]
-If you receive an unknown error, it might be due to your Certificate file containing a CA path. You must move the CA path into its own file and upload it separately before continuing.
-
-. Select *TLS Context* and then perform the following:
+All inbound traffic entering Anypoint Runtime Fabric is encrypted using TLS. When enabling inbound traffic, you must provide a valid TLS certificate. Once enabled, HTTP requests sent to Runtime Fabric receive a `301` response to redirect to HTTPS on port 443.
 +
-[NOTE]
-If you do not see *Add TLS Context*, ensure you are viewing your secret group in edit mode. Return to the *Secret Groups* page and then select *Edit* next to your group.
-
-.. Type a name for your TLS context. This will identify it when you apply it to your Runtime Fabric later.
-.. Use the default TLS version (TLS 1.2).
-.. Under *Target*, select *Anypoint Security*. This configures your TLS context so it can be applied to a Runtime Fabric.
-.. Under *Keystore*, select the name of the keystore you configured previously.
-.. Select *Save*.
-
-. After saving your TLS context, select *Secret Groups* and then select *Finish*. This enables TLS Context for your *Runtime Fabric.*
+You can configure the private key and public certificate for a TLS-enabled server in one of the following ways:
```

Comment:
```suggestion
You can configure the private key and public certificate for a TLS-enabled server by uploading either Privacy-Enhanced Mail (PEM) or Java Keystore (JKS) files or, if you are an advanced user, by importing from secrets manager:
```

Reply:
...because the three options are at quite a textual distance from one another.
Again, I think this task is overloaded with conceptual info and too deeply nested.